study-builder/studies/pancan/family-history/family-history-additional-details.conf

```diff
@@ -0,0 +1,84 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": "FAMILY_HISTORY_ADDITIONAL_DETAILS",
+  "parentActivityCode": "FAMILY_HISTORY",
+  "createOnParentCreation": true,
+  "maxInstancesPerUser": 1,
+  "displayOrder": 5,
+  "versionTag": "v1",
+  "allowUnauthenticated": false,
+  "listStyleHint": "NONE",
+
+  "translatedNames": [
+    { "language": "en", "text": ${i18n.en.family_history.additional_details.additional_details} },
+    { "language": "es", "text": ${i18n.es.family_history.additional_details.additional_details} }
+  ],
+  "translatedTitles": [
+  ],
+  "translatedSummaries": [
+
+  ],
+  "readonlyHintTemplate": null,
+  "introduction": null,
+  "closing": null,
+  "sections": [
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+                <p class="PageContent-text">$additional_p1</p>
+                <p class="PageContent-text">$additional_p2</p>
+                <p class="PageContent-text">$additional_p3</p>
+                """,
+            "variables": [
+              {
+                "name": "additional_p1",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.family_history.additional_details.p1} },
+                  { "language": "es", "text": ${i18n.es.family_history.additional_details.p1} }
+                ]
+              },
+              {
+                "name": "additional_p2",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.family_history.additional_details.p2} },
+                  { "language": "es", "text": ${i18n.es.family_history.additional_details.p2} }
+                ]
+              },
+              {
+                "name": "additional_p3",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.family_history.additional_details.p3} },
+                  { "language": "es", "text": ${i18n.es.family_history.additional_details.p3} }
+                ]
+              }
+            ]
+          },
+          "blockType": "CONTENT",
+          "shownExpr": null
+        },
+        {
+          "question": {
+            include required("../../snippets/text-question.conf"),
+            "stableId": "OTHER_FACTORS_CANCER_RISK",
+            "inputType": "ESSAY",
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "TEXT",
+              "templateText": "",
+              "variables": []
+            },
```

Comment:
This seems like a similar situation to my [other comment](https://github.com/broadinstitute/ddp-study-server/pull/1049#discussion_r751290694). There is a question, which is not part of a `COMPOSITE`, which lacks a prompt.

Reply:
This is also made on purpose. Previous blocks contains it all.
This actually is the following section:
<img width="973" alt="image" src="https://user-images.githubusercontent.com/71830706/142260781-2574a6af-9534-4e4c-9a66-4c85679fbf9d.png">

Reply:
Same thing as my other comment, this is not an acceptable design choice and needs to be reworked. A `CONTENT` block may not be used as a replacement for presenting a question's prompt.

Reply:
Moved text to the prompt